llvm/test/MC/RISCV/corev/mac-all-extensions.s

```diff
@@ -0,0 +1,98 @@
+# RUN: llvm-mc -triple=riscv32 --mattr=xcorev -show-encoding %s 2>&1 \
```

Comment:
Should this say +xcorev to match the other tests where --mattr is provided +extension